react/week3/to_do_list_with_api/src/newTask.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import DatePicker from 'react-date-picker'
+
+class CreateNewTask extends React.Component {
+    state = { ...this.props, newTaskDate: new Date(), NewTaskName: 'Add new task' }
```

Comment:
```suggestion
    state = { ...this.props, newTaskDate: new Date(), newTaskName: 'Add new task' }
```
Uppercased variable names should **only** be used for react components.